src/com/nutrons/framework/controllers/RunAtPowerEvent.java

```diff
@@ -18,10 +19,13 @@ public double power() {
 
   @Override
   public void actOn(Talon talon) {
-    if(Math.abs(power) > 1.0) {
-        throw new EventUnimplementedException("Power greater than magnitude of 1.0 is not supported for Talons");
+    talon.changeControlMode(CanControlMode.Power);
+
+    if (Math.abs(power) > 1.0) {
+      throw new EventUnimplementedException(
```

Comment:
We need an `illegalArgumentException` or an `illegalStateException` here.  This isn't quite the right thing to throw.